src/menus/menu-item.jsx

```diff
@@ -155,6 +160,21 @@ const MenuItem = React.createClass({
         React.cloneElement(secondaryText, {style: mergedSecondaryTextStyles}) :
         <div style={this.prepareStyles(styles.secondaryText)}>{secondaryText}</div>;
     }
+    let childMenuPopover;
+    if (menuItems) {
+      childMenuPopover = (
+        <Popover
+          anchorOrigin={{horizontal:'right', vertical:'top'}}
+          anchorEl={this.state.anchorEl}
+          open={this.state.open}
+          onRequestClose={this._onRequestClose}>
+          <Menu desktop={desktop} disabled={disabled} style={{position:'relative'}}>
```

Comment:
I think that we should move `{position:'relative'}` in a shared variable.
I think that we should add PureRenderMixin to the Menu component in the future.
This would prevent unnecessary rerender.